back-end/db/host.ts

```diff
@@ -134,7 +134,7 @@ export default {
   },
 
   buildQuiz: async (gameId: number, questionnaireQuestions: any, numQuizQuestions: number): Promise<IQuizQuestion[]> => {
-    const players = await playerDb.getPlayers(gameId);
+    const players = await playerDb.getPlayersWithQuestionnairesCompleted(gameId, questionnaireQuestions.length);
```

Comment:
Only want to build the quiz using the players who actually completed the questionnaire